src/stores/errors/errorsActions.ts

```diff
@@ -0,0 +1,28 @@
+import { Dispatch } from 'redux';
+
+export enum ErrorsActions {
+  RESET = '@error/reset',
+  SET = '@error/set',
+}
+
+const actionSetError = (error: unknown) => ({
+  type: ErrorsActions.SET,
+  payload: error,
+});
+const actionResetError = (errorKey: string) => ({
+  type: ErrorsActions.RESET,
+  payload: errorKey,
+});
+
+export const setError =  (
+  error: unknown,
+  dispatch: Dispatch,
+) => {
+  dispatch(actionSetError(error));
+};
+
+export const resetErrors =  (
+  dispatch: Dispatch, errorKey: string,
```

Comment:
dispatch should always be at the same position to avoid ambiguity. So put it into the end